core-service/app/Models/Course.php

```diff
@@ -26,21 +27,27 @@
  */
 class Course extends EgalModel
 {
-  use HasFactory;
+	use HasFactory;
 
-  protected $fillable = [
-    "title",
-    "student_capacity",
-    "start_date",
-    "end_date",
-    "has_certificate",
-  ];
-  protected $guarder = [
-    "id"
-  ];
+	protected $fillable = [
+		"title",
+		"student_capacity",
+		"start_date",
+		"end_date",
+		"has_certificate",
+	];
+	protected $guarder = [
+		"id"
+	];
 
-  public function lessons()
-  {
-    return $this->hasMany(Lesson::class);
-  }
+	public function lessons(): HasMany
+	{
+		return $this->hasMany(Lesson::class);
+	}
+
+	public static function findItem(int $id): mixed
+	{
+		$query = new static();
```

Comment:
второй вариант, без создания instance, 
```suggestion
		return self::...
```